Pipfile

```diff
@@ -4,36 +4,70 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+annotated-types = "==0.7.0"
+attrs = "==24.2.0"
+boto3 = "==1.26.140"
+botocore = "==1.29.165"
+cachetools = "==5.5.0"
+certifi = "==2024.8.30"
+charset-normalizer = "==3.4.0"
+click = "==8.1.7"
+deepdiff = "==6.4.1"
+future = "==1.0.0"
+gitdb = "==4.0.11"
+gitpython = "==3.1.43"
+google-auth = "==2.35.0"
+idna = "==3.10"
+jmespath = "==1.0.1"
 jsonschema = "==4.23.0"
-mypy = "==1.11.0"
-"ruamel.yaml" = "==0.18.6"
-kubernetes = "==27.2.0"
+jsonschema-specifications = "==2024.10.1"
 junitparser = "==2.8.0"
-pyjwt = "==2.5.0" # forced downgrade due to https://stackoverflow.com/questions/33198428/jwt-module-object-has-no-attribute-encode
-python-on-whales = "==0.72.0"
-gitpython = "==3.1.43"
-click = "==8.1.7"
-rich = "==13.8.0"
+kubernetes = "==27.2.0"
+markdown-it-py = "==3.0.0"
+mdurl = "==0.1.2"
+mypy = "==1.11.0"
+mypy-extensions = "==1.0.0"
+oauthlib = "==3.2.2"
+ordered-set = "==4.1.0"
 pyaml-env = "==1.2.1"
+pyasn1 = "==0.6.1"
+pyasn1-modules = "==0.4.1"
+pydantic = "==2.9.2"
+pydantic-core = "==2.23.4"
+pygments = "==2.18.0"
+pyjwt = "==2.5.0"
+python-dateutil = "==2.9.0.post0"
 python-dotenv = "==1.0.0"
-toml = "==0.10.2"
-boto3 = "==1.26.140"
-deepdiff = "==6.4.1"
-requests = "==2.31.0"
+python-on-whales = "==0.72.0"
+pyyaml = "==6.0.2"
 referencing = "==0.35.1"
+requests = "==2.31.0"
+requests-oauthlib = "==2.0.0"
+rich = "==13.8.0"
+rpds-py = "==0.20.0"
+rsa = "==4.9"
+"ruamel.yaml" = "==0.18.6"
+"ruamel.yaml.clib" = "==0.2.12"
+s3transfer = "==0.6.2"
+shellingham = "==1.5.4"
+six = "==1.16.0"
+smmap = "==5.0.1"
+toml = "==0.10.2"
+tqdm = "==4.66.5"
+typer = "==0.12.5"
+typing-extensions = "==4.12.2"
+urllib3 = "==1.26.20"
+websocket-client = "==1.8.0"
```

Comment:
Where do all these extra packages come from? We didn't need them before, why now

Reply:
I regenerated both `Pipfile` and `Pipfile.lock` automatically from `requirements.txt`, and [these packages are defined there](https://github.com/Vandebron/gh-mpyl/blob/7667be9d02d80e3e24599f4f21ac36cb2548bc15/requirements.txt#L52-L55). Why they were added to `requirements.txt` I don't know 🙂

Reply:
I also just realized that if you created `requirements.txt` from `Pipfile` then there's no point in doing the other way around, so I reverted those changes (and kept only the removal of the unnecessary scripts).

Reply:
It's weird that when you generate pipfile -> requirements -> pipfile you end up with a different pipfile, but whateverrrr